skimage/transform/tests/test_warps.py

```diff
@@ -517,3 +520,78 @@ def test_zero_image_size():
     with testing.raises(ValueError):
         warp(np.zeros((10, 10, 0)),
              SimilarityTransform())
+
+
+def test_linear_polar_mapping():
```

Comment:
This looks like an excellent use case for hypothesis.

https://hypothesis.readthedocs.io/en/latest/

Reply:
Unfortunately we are not ready to add hypothesis as a test dependency yet, although this is something we could consider in the future. I don't think we should saddle this PR with a new dependency, which will only slow down its time to merge.